### PR TITLE
postgresql_ping: improve documentation

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible-collections/community.postgresql/issues/174

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: plugins/modules/postgresql_ping.py [hunk 1]
```diff
@@ -51,14 +51,27 @@
 
 # In the example below you need to generate certificates previously.
 # See https://www.postgresql.org/docs/current/libpq-ssl.html for more information.
-- name: PostgreSQL ping dbsrv server using not default credentials and ssl
+- name: >
+    Ping PostgreSQL server using not default credentials and SSL,
+    register return values into the result variable for future use
```
Comment (hunleyd) [inline plugins/modules/postgresql_ping.py:56]:
```suggestion
    Ping PostgreSQL server using non-default credentials and SSL
    registering the return values into the result variable for future use
```

File: plugins/modules/postgresql_ping.py [hunk 2]
```diff
@@ -51,14 +51,27 @@
...
     ca_cert: /root/root.crt
     ssl_mode: verify-full
+  register: result
+
+- name: Assert that the server is available, fail otherwise
+  assert:
+    that:
+    - result.is_available == yes
+
+# You can use the variable with another task
```
Comment (hunleyd) [inline plugins/modules/postgresql_ping.py:71]:
```suggestion
# You can use the registered result with another task
```

**Review by hunleyd — CHANGES_REQUESTED** [review]

**Comment by Andersson007** [comment]
@hunleyd , committed the suggestions, now the comments are much better! thanks!:)

**Comment by Andersson007** [comment]
PTAL

**Review by hunleyd — APPROVED** [review]

**Comment by Andersson007** [comment]
@hunleyd thanks for reviewing!

**Pull request merged**